mobile/src/main/java/org/hawkular/client/android/fragment/ConfirmOperationFragment.java

```diff
@@ -0,0 +1,255 @@
+                operation.getOperationProperties().getOperationParameters() == null ||
+                operation.getOperationProperties().getOperationParameters().size() == 0
+                ) {
+            custom.setEnabled(false);
+        }
+
+        execute.setOnClickListener(new ClickListner());
+        cancel.setOnClickListener(new ClickListner());
+
+        custom.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
+            @Override public void onCheckedChanged(CompoundButton compoundButton, boolean checked) {
+                if (checked) {
+                    scroll.setVisibility(View.VISIBLE);
+                } else {
+                    scroll.setVisibility(View.GONE);
+                }
+            }
+        });
+    }
+
+    public void setCallback(Callback<String> callback) {
+        this.callback = callback;
+    }
+
+    private void setOperationDetail() {
+        operationDetail.setText(operation.getName() + " on " + resource.getName());
+    }
+
+    private Resource getResource() {
+        return getArguments().getParcelable(Fragments.Arguments.RESOURCE);
+    }
+
+    private Operation getOperation() {
+        return getArguments().getParcelable(Fragments.Arguments.OPERATION);
+    }
+
+    private void createTable() {
+        TableRow row;
+        List<OperationParameter> operationParameters = operation.getOperationProperties().getOperationParameters();
+
+        if (operationParameters != null) {
+            for (OperationParameter operationParameter : operationParameters) {
+                switch (operationParameter.getType()) {
+                    case "string": {
+                        row = (TableRow) LayoutInflater.from(getActivity()).inflate(R.layout.row_edit, null);
+                        EditText data = (EditText) row.findViewById(R.id.data);
+                        data.setText(operationParameter.getDefaultValue());
+                        data.setInputType(InputType.TYPE_CLASS_TEXT);
+                        break;
+                    }
+
+                    case "int": {
+                        row = (TableRow) LayoutInflater.from(getActivity()).inflate(R.layout.row_edit, null);
+                        EditText data = (EditText) row.findViewById(R.id.data);
+                        data.setText(operationParameter.getDefaultValue());
+                        data.setInputType(InputType.TYPE_CLASS_NUMBER);
+                        break;
+                    }
+
+                    case "float": {
+                        row = (TableRow) LayoutInflater.from(getActivity()).inflate(R.layout.row_edit, null);
+                        EditText data = (EditText) row.findViewById(R.id.data);
+                        data.setText(operationParameter.getDefaultValue());
+                        data.setInputType(InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_FLAG_DECIMAL);
+                        break;
+                    }
+
+                    default: {
```

Comment:
I guess this is meant for boolean. Better have this explicit as `case "boolean":` and use the default for cases where the incoming data is not known. As is it works, but it is not obvious when you read the code.